lib/tasks/dev.rake

```diff
@@ -0,0 +1,169 @@
+desc 'Fill the database tables with some sample data'
+task({ sample_data: :environment }) do
+  starting = Time.now
+  p "Creating sample data..."
+
+  Match.delete_all
```

Comment:
could potentially put all of this into 
```ruby
if Rails.env.development?
  Match.delete_all
  CohortMember.delete_all
  ProgramAdmin.delete_all
  Cohort.delete_all
  Program.delete_all
  User.delete_all
end
```
could helps prevent unintended data loss in production